packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -774,7 +775,10 @@ export default class Meetings extends WebexPlugin {
           EVENT_TRIGGERS.MEETINGS_REGISTERED
         );
         this.registered = true;
-        Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.MEETINGS_REGISTRATION_SUCCESS);
+        Metrics.sendBehavioralMetric(
+          BEHAVIORAL_METRICS.MEETINGS_REGISTRATION_SUCCESS,
+          metricFields
```

Comment:
maybe better to have some sanity check on `metricFields` in case some app accidentally passes some huge object here? although not sure if it's worth it, because if they want to be malicious they can always call webex.internal.metrics.submitClientMetrics directly and send whatever they want

Reply:
it terms of functionality i don't see a issue , if someone wants to bombad the service they can by checking our network tab

only thing I am concerned is the documentation should show up in metricsField and it might confuse the developers. do you want to change the name metricFields to _metricFields or something which does make them to pass anything